src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -328,7 +328,7 @@ Modelers:
     <value>Parameters "subscriptionId" and "api-version" are not allowed in the operations section</value>
   </data>
   <data name="ServiceDefinitionParametersMissingMessage" xml:space="preserve">
-    <value>Parameters "subscriptionId" or "api-version" are referenced but not defined in the parameters section of Service Definition</value>
+    <value>Parameters "subscriptionId" and "api-version" must be defined in the parameters section in the swagger for the Azure Management Plane service.</value>
```

Comment:
Could you please point me to the rule that validates that they must be in global parameter section? 

Based on rule id [M2014](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) seems like this rule supposed to do that but current implementation doesn't differentiate.